File: VetAPI/Models/Prontuario.cs
namespace VetAPI.Models;

public class Prontuario
{
    public int ProntuarioId { get; set; }
    public Animal? Animal { get; set; }
    public DateTime Data { get; set; }
    public List<EstadoDoAnimal>? EstadosAnimais { get; set; }
    public List<Medicamento>? Medicamentos { get; set; }

}

File: VetAPP/src/app/Prontuario.ts
import { EstadoDoAnimal } from './EstadoDoAnimal';
import { Medicamento } from './Medicamento';

export class Prontuario{
    prontuarioId: number = 0;
    data: Date | undefined;
    estadoDoAnimal: EstadoDoAnimal[] | undefined;
    medicameto: Medicamento[] | undefined;
}